R/install.R

```diff
@@ -436,8 +442,8 @@ renv_install_package_cache <- function(record, cache, linker) {
   after <- Sys.time()
 
   type <- case(
-    identical(linker, renv_file_copy) ~ "copied from cache",
-    identical(linker, renv_file_link) ~ "linked from cache"
+    identical(linker, renv_file_copy) ~ "copied cache",
```

Comment:
IMO it's worth keeping the from to more clearly distinguish to copying to the cache.